src/bbmustache.erl

```diff
@@ -689,3 +707,73 @@ is_recursive_data(_)                                -> false.
 is_recursive_data([Tuple | _]) when is_tuple(Tuple) -> true;
 is_recursive_data(_)                                -> false.
 -endif.
+
+%% @doc Processes command-line commands (render, ...)
+-spec process_commands([getopt:option()], [string()]) -> ok.
+process_commands([], []) ->
+    help();
+process_commands([help | _], _) ->
+    help();
+process_commands([version | _], _Options) ->
+    {ok, AppConfig} = application:get_all_key(bbmustache),
+    Version = proplists:get_value(vsn, AppConfig),
+    io:format("~p~n", [Version]);
```

Comment:
These options should be a high priority, regardless of order.

```erl
HasHelp = proplists:is_defined(help, Opts),
HasVersion = proplists:is_defined(version, Opts),
if
    HasHelp                  -> print_help();
    HasVersion               -> print_version();
    Opts =:= [], Cmds =:= [] -> print_help();
    Cmds =:= []              -> ...; % no files
    true ->
         .... % others
end
```

Reply:
f551d29a39d620da2886959b5731a2da1e5481c3